.github/workflows/publish-deps.yml

```diff
@@ -26,29 +26,21 @@ jobs:
         with:
           fetch-depth:                              5
           submodules:                               recursive
-      - name:                                       Set vars
-        id:                                         vars
-        run:                                        |
-          if [[ ${GITHUB_REF} = refs/tags/* ]]
-          then
-            echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
-          fi
       - name:                                       Build and push dependencies
         uses:                                       docker/build-push-action@v1
         with:
           username:                                 ${{ secrets.DOCKER_USER }}
           password:                                 ${{ secrets.DOCKER_PASSWORD }}
-          repository:                               paritytech/parity-bridges-common
+          repository:                               paritytech/bridge-dependencies
```

Comment:
Is this supposed to be up on Docker Hub already?

Reply:
The `paritytech/parity-bridges-common` image was not (I forgot to change the name). However, the `paritytech/bridge-dependencies` is since I published it to make sure that this Action was working correctly